spec/unleash/constraint_spec.rb

```diff
@@ -20,12 +20,6 @@
 
       constraint = Unleash::Constraint.new('env', 'IN', ['dev', 'pre'])
       expect(constraint.matches_context?(context)).to be true
-
-      constraint = Unleash::Constraint.new('env', 'NOT_IN', ['dev', 'pre'])
-      expect(constraint.matches_context?(context)).to be false
-
-      constraint = Unleash::Constraint.new('env', 'NOT_IN', ['pre', 'prod'])
-      expect(constraint.matches_context?(context)).to be true
```

Comment:
what's the rationale behind removing these tests?

Reply:
These are actually exact replicas of two tests further down in the file. Pretty sure this was originally me with fat ctrl-v fingers, sorry!,

Reply:
good catch!

Reply:
Thinking about it twice, maybe we can move this clean up snippet to a separate trivial clean up PR?

Reply:
would it be ok to put this in a separate PR, just so we have one PR per problem / issue getting fixed?

Reply:
ok.